wiki/en/Software-Manual.md

```diff
@@ -259,3 +273,41 @@ Attempts to detect audio feedback loops. Once detected, this feature will activa
 Controls the relative levels of the left and right local audio channels. For a mono signal
 it acts as a pan between the two channels. For example, if a microphone is connected to the right input channel and
 an instrument is connected to the left input channel which is much louder than the microphone, move the audio fader to increase the relative volume of the mic.
+
+# Menu commands
+
+### File > Load/Save Mixer Channels Setup
+ 
+You can save and restore the mix you have for your band rehearsals (fader, mute, pan, solo etc.) and load these any time (even while you are playing). Loading can also be done by drag/drop to the mixer window.
+
+
+### Edit > Auto-Adjust All Faders 
+
+Applies a one-off fader setting to each channel depending on its volume. Useful for large ensembles to get a reasonable overall mix, although individual adjustments might still be necessary. Best applied during a warm-up or a uniform part of the music piece. 
+
+# Command Line Options
+
+Most common functions in Jamulus can be set using the GUI, but these and others can also be set using options given in a terminal window. Exactly how you do this will depend on your operating system.
+
+For example on Windows, to use a specific settings file, right-click on the Jamulus shortcut and choose "Properties" > Target. Add the necessary arguments to Jamulus.exe:
+
+```shell
+"C:\Program Files (x86)\Jamulus\Jamulus.exe" --inifile "C:\path\to\myinifile.ini"
```

Comment:
"C:\Program Files (x86)\Jamulus\Jamulus.exe" doesn't exist anymore with new versions. Should be "C:\Program Files\Jamulus\Jamulus.exe"